iambic/plugins/v0_1_0/aws/iam/policy/template_generation.py

```diff
@@ -330,7 +367,7 @@ async def create_templated_managed_policy(  # noqa: C901
         AwsIamManagedPolicyTemplate,
         template_params,
         ManagedPolicyProperties(**template_properties),
-        list(aws_account_map.values()),
+        [aws_account_map[mp_ref["account_id"]] for mp_ref in managed_policy_refs],
```

Comment:
this broke this functional test: FAILED functional_tests/aws/managed_policy/test_template_generation.py::PartialImportManagedPolicyTestCase::test_delete_managed_policy_from_one_account